cpg_workflows/stages/gatk_sv/gatk_sv_multisample_2.py

```diff
@@ -568,7 +568,7 @@ def expected_outputs(self, cohort: Cohort) -> dict[str, Path]:
     def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput:
 
         # allow for no prior name/IDs
-        if spicy_vcf := query_for_spicy_vcf(cohort.analysis_dataset.name) is None:
+        if (spicy_vcf := query_for_spicy_vcf(cohort.analysis_dataset.name)) is None:
```

Comment:
Absolutely can... I don't remember why I did it this way...

Reply:
Ahh I remember now. You can't do 
```
if not spicy_vcf := query_for_spicy_vcf(cohort.analysis_dataset.name):
    ...
```

because `spicy_vcf` isn't defined yet. You can do 

```
if not (spicy_vcf := query_for_spicy_vcf(cohort.analysis_dataset.name)):
    ...
```

which is the best of both worlds